apps/generator/lib/generator.js

```diff
@@ -604,11 +605,9 @@ class Generator {
         saveType: 'prod',
         save: false
       });
-
-      const addResult = arb[Symbol.for('resolvedAdd')];
-      if (!addResult) throw new Error('Unable to resolve the name of the added package. It was most probably not added to node_modules successfully');
-
-      const packageName = addResult[0].name;
+      const manifest = await pacote.manifest(this.templateName);
+      const packageName = manifest.name;
+      
```

Comment:
we need more `log.debug`, no other way 🤷🏼 

for now, dummy, no need to put them in `logMessages`

```suggestion
      log.debug(`Completed installation of template represented as: ${this.templateName}`);
      const manifest = await pacote.manifest(this.templateName);
      const packageName = manifest.name;
      log.debug(`Actual name of installed template is: ${packageName}`);
```